### ISSUE TITLE
Custom Field Error 500 - Can't delete or edit custom fields

### ISSUE THREAD

**Issue by p-mac90** [grokability/snipe-it#10276]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

It's not possible to delete or edit a custom field. Error 500 appears on the website in different browser. 

I suspect that some custom field cause an another error in snipe-it i get.  

### Reproduction steps

1. Open custom field menu
2. press on delete button or save the edit 
3. Error 500 appears
...

### Expected behavior

Custom field's should be deleted or edit

### Screenshots

![Bildschirmfoto 2021-11-05 um 11 20 59](https://user-images.githubusercontent.com/40386292/140495881-377f3e40-8acb-46d8-98af-0a602c04a796.jpg)

### Snipe-IT Version

5.3.0

### Operating System

Ubuntu

### Web Server

Apache

### PHP Version

7.4.3

### Operating System

Mac

### Browser

Chrome, Firefox

### Error messages

```shell
SQLSTATE[42000]: Syntax error or access violation: 1091 Can't DROP COLUMN `_snipeit_kundennummer_17`; check that it exists (SQL: alter table `assets` drop `_snipeit_kundennummer_17`)
Previous exceptions
SQLSTATE[42000]: Syntax error or access violation: 1091 Can't DROP COLUMN `_snipeit_kundennummer_17`; check that it exists (42000)
SQLSTATE[42000]: Syntax error or access violation: 1091 Can't DROP COLUMN `_snipeit_kundennummer_17`; check that it exists (42000)
```

### Additional context

/var/www/snipeit/vendor/laravel/framework/src/Illuminate/Database/Connection.php
     * @param  array  $bindings
     * @param  \Closure  $callback
     * @return mixed
     *
     * @throws \Illuminate\Database\QueryException
     */
    protected function runQueryCallback($query, $bindings, Closure $callback)
    {
        // To execute the statement, we'll simply call the callback, which will actually
        // run the SQL against the PDO connection. Then we can calculate the time it
        // took to execute and log the query SQL, bindings and time in our memory.
        try {
            $result = $callback($query, $bindings);
        }
 
        // If an exception occurs when attempting to run a query, we'll format the error
        // message to include the bindings with SQL, which will make this exception a
        // lot more helpful to the developer instead of just the database's errors.
        catch (Exception $e) {
            throw new QueryException(
                $query, $this->prepareBindings($bindings), $e
            );
        }
 
        return $result;
    }
 
    /**
     * Log a query in the connection's query log.
     *
     * @param  string  $query
     * @param  array  $bindings
     * @param  float|null  $time
     * @return void
     */
    public function logQuery($query, $bindings, $time = null)
    {
        $this->event(new QueryExecuted($query, $bindings, $time, $this));
 
        if ($this->loggingQueries) {
Arguments
"SQLSTATE[42000]: Syntax error or access violation: 1091 Can't DROP COLUMN `_snipeit_kundennummer_17`; check that it exists (SQL: alter table `assets` drop `_sni ▶"

**Comment by p-mac90** [comment]
Problem solved 

I had three custom fields without a fieldset that caused the problem. I deleted them directly from the database now the search works again.

**Comment by nzamanie** [comment]
Thanks to this ticket we were able to fix our issue.

We had the same issue with custom fields.  We decided to delete the last 3 entry into the database.  

**OS: Ubuntu 22.04  
Snipe-IT versionv6.0.13 build 8923 (g30f4af3ac)  
Software License[AGPL3]  
PHP Version8.1.2-1ubuntu2.9  
Laravel Version8.83.22**  

![image](https://user-images.githubusercontent.com/119643032/205202582-024a3e7b-8a68-4ffd-a50c-4e6a5409ee65.png)
